winterfell/webapp/src/js/net.js

```diff
@@ -1,12 +1,26 @@
 'use strict';
 var app = angular.module('vetafiApp');
 app.factory('net', ['$http', function($http) {
-  var localDevHost = 'localhost:3000';
-  var baseUrl = "http://localhost:3999";
+  var localFrontDevHost = 'localhost:3000';
+  var baseUrl;
+  var dev = true; // todo this should be determined by a gulp task
+  if (dev) {
+    baseUrl = "http://localhost:3999";
+  } else {
+    baseUrl = "https://www.vetafi.org";
+  }
+
+  // social properties
+  var socialIdMe = {
+    clientId: '71ffbd3f04241a56e63fa6a960fbb15e',
+    redirectServerUri: baseUrl + "/auth/link/idme",
```

Comment:
You don't need to include the base url (https://www.vetafi.org) in the request,  the AJAX default is to resolve the path relative to the current page.. so this could be just `redirectServerUri: "/auth/link/idme"` and the above logic about what the base url is in dev vs prod can be safely removed